services/proxy/image.go

```diff
@@ -21,5 +21,5 @@ func PullImages() {
 		return
 	}
 
-	ComposeStdout("pull")
+	composeStdout("pull")
```

Comment:
A lot here going exported to non-exported, I figured we'd use them cross package. But probably won't need to